src/plugins/home/public/application/components/home.js

```diff
@@ -161,35 +160,28 @@ export class Home extends Component {
       <Fragment>
         <header className="homPageHeader">
           <EuiFlexGroup gutterSize="none">
-            <EuiFlexItem>
-              <EuiScreenReaderOnly>
+            <EuiFlexItem className="homPageHeader__title">
+              <EuiTitle size="m">
                 <h1>
-                  <FormattedMessage id="home.welcomeHomePageHeader" defaultMessage="Kibana home" />
+                  <FormattedMessage
+                    id="home.pageHeader.welcomeNoUserTitle"
+                    defaultMessage="Welcome to {ELASTIC}!"
+                    values={{ ELASTIC: 'Elastic' }}
+                  />
                 </h1>
-              </EuiScreenReaderOnly>
-              <EuiFlexGroup gutterSize="none">
-                <EuiTitle size="m">
-                  <h1>
-                    <FormattedMessage
-                      id="home.pageHeader.welcomeNoUserTitle"
-                      defaultMessage="Welcome to {ELASTIC}!"
-                      values={{ ELASTIC: 'Elastic' }}
-                    />
-                  </h1>
-                </EuiTitle>
-              </EuiFlexGroup>
+              </EuiTitle>
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <EuiFlexGroup alignItems="flexEnd">
-                <EuiFlexItem>
+              <EuiFlexGroup className="homPageHeader__menu" alignItems="flexEnd">
```

Comment:
As it stands, the menu stays horizontal down to tablet portrait size. It only stacks below 768, however if set to `responsive={false}` then the menu goes off the screen. I'd like to convert this to a button + menu/popover, but then I wonder if we should consider hiding the menu altogether since doing these things on a phone is unrealistic to begin with. Let's leave it as-is, its functional, and consider improving it later if there is sufficient demand for further investment of time.